src/core/MythosArea.ttslua

```diff
@@ -35,13 +36,37 @@ end
 function onLoad(savedData)
   if savedData and savedData ~= "" then
     local loadedState = JSON.decode(savedData) or {}
+    scenarioCard      = getObjectFromGUID(loadedState.scenarioCardGUID) or {}
```

Comment:
this should not default to a table, just keep it `nil` and handle that accordingly where it matters